src/components/auth/Connect.svelte

```diff
@@ -1,3 +1,23 @@
+<script lang="ts">
+  import { goto } from '$app/navigation'
+  import { onDestroy, onMount } from 'svelte'
+
+  import { sessionStore } from '../../stores'
+  import type { Session } from '$lib/session'
+
+  let unsubscribeSessionStore: () => void = () => {}
+
+  onMount(() => {
+    unsubscribeSessionStore = sessionStore.subscribe((session: Session) => {
+      if (session.authed) {
+        goto('/')
+      }
+    })
+  })
```

Comment:
@bgins Btw, I read this in the [Svelte Tutorial](https://svelte.dev/tutorial/onmount): 

> If the `onMount` callback returns a function, that function will be called when the component is destroyed.

So, it's less explicit, but couldn't you return `unsubscribeSessionStore` from `onMount` and it would be automatically invoked, unsubscribing the session?